Longest_Common_Subsequence/Longest_Common_Subsequence.cpp

```diff
@@ -0,0 +1,38 @@
+/* Problem Statement- Given two sequences, find the length of longest subsequence present in both of them. A subsequence is a sequence that can be derived from another sequence by deleting some or no elements without changing the order of the remaining elements.*/
+#include<bits/stdc++.h> 
+using namespace std;
+
+// Function to calculate length of longest common subsequence
+int LongestCommonSubsequence(int arr1[], int arr2[], int n, int m) 
```

Comment:
Please use size1 and size2 instead of n and m.

Reply:
changed variable names